cfgov/jinja2/v1/_includes/organisms/table.html

```diff
@@ -12,35 +12,40 @@
 
    ========================================================================== #}
 
+{% if value.data  %}
 <table class="o-table
               {{ 'o-table__row-links' if value.row_links else '' }}
-              table__stack-on-small
-              table__entry-header-on-small">
+              {{ 'table__stack-on-small' if value.is_stacked else '' }}
+              {{ 'table__striped' if value.is_striped else '' }}
+              {{ 'u-w100pct' if value.is_full_width else '' }}">
+    {% set row_index = 0 %}
+    {% if value.first_row_is_table_header and value.data | list | length > 0 %}
+    {% set row_index = 1 %}
     <thead>
         <tr>
-            {% for thead in value.headers %}
-                <th>{{ thead }}</th>
+            {% for column in value.data[0] %}
+                {% if cell is not none %}
+                <th>
```

Comment:
Add `scope="col"` to these `th` tags. It'll [help screenreaders](http://webaim.org/techniques/tables/data#scope).